module.txt

```diff
@@ -1,12 +1,14 @@
 {
     "id": "InGameHelp",
     "version": "1.1.0",
-    "author": "Josharias, J0hnCena, Cervator, xrtariq2594, msteiger",
+    "author": "The Terasology Foundation",
     "displayName": "In-Game Help",
-    "description": "Adds a in-game help window primarily for items (what it does, what it is, etc.)",
+    "description": "Adds an in-game help window primarily for items (what it does, what it is, etc.)",
     "dependencies": [
+        { "id": "CoreAssets", "minVersion": "2.0.0" },
         { "id": "InGameHelpAPI", "minVersion": "1.0.0" },
-        { "id": "Inventory", "minVersion": "1.1.0" }
+        { "id": "Inventory", "minVersion": "1.1.0" },
+        { "id": "Notifications", "minVersion": "0.1.0", "maxVersion": "0.3.0" }
```

Comment:
Hm, I don't like this, but it would get quite hairy for the local dev setup otherwise 🧐 

**The problem:** Notifications is published in v0.1.0 (`master` branch state) and snapshots are published for v0.2.0-SNAPSHOT (`develop` branch). Since we are in the v0.x version range, this is treated as major version, and thus excluded from the default dependency range.

I suspect this would also break when blindly building the Omega line-up which would choose the latest SNAPSHOT version...